views/views.py

```diff
@@ -88,6 +96,85 @@ def partial_update(self, request, *args, **kwargs):
                     },
                     status=status.HTTP_403_FORBIDDEN,
                 )
+
+        """ Notifications start """
+
+        request_person = self.request.person
+        query_id = request.data.get('id')
+        query = Query.objects.get(id=query_id)
+        category_obj = get_base_category()
+        url = f'/helpcentre/issue/{query_id}'
+
+        # For query status related notification
+        if 'is_closed' in request.data.keys():
+            if query.is_closed != request.data['is_closed']:
+                if request.data['is_closed'] is True:
+                    status = 'closed'
+                else:
+                    status = 'open'
+                template = f'Your query {query.title} has been marked {status}'
+                person_id = query.uploader.id
+                push_notification(template=template, category=category_obj,
+                                  web_onclick_url=url,
+                                  person=person_id, is_personalised=True)
+
+        # For assignee related notification
+        if 'assignees' in request.data.keys():
+            new_assignees = request.data['assignees']
+            old_assignees = list(query.assignees.all().values_list(
+                'id', flat=True))
+            is_added = None
+            assignee_id = None
+            for i in new_assignees:
+                if assignee_id is not None:
+                    break
+                if i not in old_assignees:
+                    is_added = True
+                    assignee_id = i
+                    break
+            for i in old_assignees:
+                if assignee_id is not None:
+                    break
+                if i not in new_assignees:
+                    is_added = False
+                    assignee_id = i
+                    break
+            status = "assigned to" if is_added is True else "unassigned from"
+            common_assignees = [i for i in new_assignees if i in old_assignees]
+
+            # For the fellow assignees
+            person = Maintainer.objects.get(id=assignee_id).person
+            person_ids = [Maintainer.objects.get(
+                id=i).person_id for i in common_assignees]
```

Comment:
Why did you break line here, `Maintainer.objects.get(id=i).person_id` would be fine on one line.